pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -211,59 +215,152 @@ def default_model_config(self) -> Dict[str, Dict]:
             "alpha_prior": {"dist": "Weibull", "kwargs": {"alpha": 2, "beta": 10}},
             "s_prior": {"dist": "Weibull", "kwargs": {"alpha": 2, "beta": 1}},
             "beta_prior": {"dist": "Weibull", "kwargs": {"alpha": 2, "beta": 10}},
+            # TODO: Cauchy prior is a bit too extreme? Probably not needed if parameters are standardized
+            "dropout_covariates_prior": {
+                "dist": "StudentT",
+                "kwargs": {"nu": 1},
+            },
+            "purchase_covariates_prior": {
+                "dist": "StudentT",
+                "kwargs": {"nu": 1},
```

Comment:
Did you say "kwargs": {"nu": 2}, is needed for data to be centered at 0? For these covariate coefficients to be interpretable, covariates need to be scaled between 0 and 1, with a prior supporting negative values. `StudentT slows down model fitting considerably, so I'm open to alternatives.